js/script.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+let toggleMenu;
+window.onload = (event) => {
+  toggleMenu = false;
+};
+
+
+const menuIcon = document.querySelector('.nav__menu-line');
+let landingPage = document.querySelector('.landing-page');
+let navBackground = document.createElement('div');
+navBackground.id = 'navBackground';
+navBackground.className = 'background';
+
+let landing = document.querySelector('.landing');
+
+let menuLineWrapper = document.querySelector('.nav__menu-line');
+let menuLine = document.querySelectorAll('.nav__menu:not(nav__menu--1)');
+
+let menuBox = document.querySelector('.nav__menu-box');
+let menuItem = document.querySelector('.nav__menu-item');
+let menuLinks = document.querySelectorAll('.nav__menu-link');
+let navLogo = document.querySelector('.nav__logo');
+
+// close icon
+let menuCloseIcon = document.getElementById('menuCloseIcon');
+
+menuIcon.addEventListener('click', function() {
+  if(!toggleMenu) {
+    onOpenMenu();
+    toggleMenu = true;
+  }
+});
+
+menuCloseIcon.addEventListener('click', function() {
+  if(toggleMenu) {
+    onCloseMenu();
+    toggleMenu = false;
+  }
+});
+
+menuLinks.forEach(menuLink => {
+  menuLink.addEventListener('click', function() {
+    if(toggleMenu) {
+      onCloseMenu();
+      toggleMenu = false;
+    }
+  });
+});
+
+// open menu
+let onOpenMenu = function() {
+  landingPage.prepend(navBackground);
+  menuCloseIcon.style.display = 'inline-block';
+  landing.classList.add('filtertext');
+  navLogo.classList.add('filtertext');
+  menuLineWrapper.style.display = 'none';
+
+  menuBox.style.display = 'flex';
+  menuBox.classList.add('nav__menu-mobile');
+
+  menuLinks.forEach(menuLink => {
+    menuLink.classList.remove('nav__menu-link');
+    menuLink.classList.add('nav__menu-mobile-item');
+  });
+}
+
+let onCloseMenu = function() {
+  landingPage.removeChild(navBackground);
+  menuCloseIcon.style.display = 'none';
+  landing.classList.remove('filtertext');
+  navLogo.classList.remove('filtertext');
+  menuLineWrapper.style.display = 'block';
+  menuBox.style.display = 'none';
+  menuBox.classList.remove('nav__menu-mobile');
+  menuLinks.forEach(menuLink => {
+    menuLink.classList.remove('nav__menu-mobile-item');
+    menuLink.classList.add('nav__menu-link');
+  });
```

Comment:
Hi @SirriCelles 
Your app works very well, just one important issue you need to fix.
Please add a cross(X) button to close the hamburger menu.
Please kindly fix this issue if not the user will find it difficult to return to the home page after clicking on the `See Project` button 
![harmbaer](https://user-images.githubusercontent.com/55596970/167877589-102023ae-c1f9-497c-b70c-d5828ace2da9.png)